authapi/urls.py

```diff
@@ -40,4 +40,7 @@
 urlpatterns = [
     url(r'^', include(router.urls)),
     url(r'^/user/tokens/$', views.TokenView.as_view(), name='create-token'),
+    url(
+        r'^/user/permissions/$', views.UserPermissionsView.as_view(),
+        name='get-user-permissions')
```

Comment:
Didn't we decide to have a `/user` endpoint that contains the permissions as a property (and the `admin` property)? Or are we going to add that separately?